dashboards/grafana-dashboard-kessel-inventory-api.configmap.yaml

```diff
@@ -2125,15 +2088,14 @@ data:
         ]
       },
       "time": {
-        "from": "now-15m",
+        "from": "now-6h",
```

Comment:
slight nit, can we keep the dashboards roughly the same time offset as each other? 
"now-15m"?